source/services/dataContracts/converters/aliasConverter/aliasConverter.ts

```diff
@@ -12,6 +12,7 @@ export class AliasConverter<TDataType> implements IConverter<TDataType> {
 		}
 
 		raw = parent[this.alias];
+		parent[this.alias] = null;
```

Comment:
Reshaping objects in JS is expensive, but we may want to consider:
```
delete parent[this.alias];
```

As this would leave the property undefined.